roles/katuma_reports/templates/katuma_reports.conf.j2

```diff
@@ -1,3 +1,3 @@
-location /admin/reports {
+location {{ ofn_route_override }} {
```

Comment:
I might be missing something but this new path wouldn't match the requests for the assets and so they won't be found.

Reply:
I think we need another location `/admin/reports/assets` block that serves only assets

Reply:
Calla